joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/passes/AstCreator.scala

```diff
@@ -738,15 +738,6 @@ class AstCreator(fileWithMeta: KtFileWithMeta, xTypeInfoProvider: TypeInfoProvid
         .withChildren(membersFromPrimaryCtorAsts)
         .withChildren(secondaryConstructorAsts)
         .withChildren(componentNMethodAsts.toList)
-        // TODO: reenable initializer block parsing when methodReturn nodes have been added
```

Comment:
I'm assuming this comment isn't relevant anymore

Reply:
it's not, that's why i deleted it : )